README.md

```diff
@@ -728,6 +728,30 @@ In either case:
       end
     ```
 
+* <a name="unless-with-comparator"></a>Avoid `unless` with comparators if you can use `if` with an opposing comparator.<sup>[[link](#unless-with-comparator)]</sup>
```

Comment:
Ok, I wasn't in that discussion, but a brief rationale is fine. Especially if you link to a source somewhere where it is explained in greater detail. I would like to read the explanation myself frankly.